Make sv_port read only

ChillerDragon opened the pull request:
The same could probably be done for econ configs and ``bindaddr``. But I am not 100% confident about them. But changing sv_port and it never having an effect is confusing and it also should never have an effect. Having a ``reload`` read that in would be horrible ux for connected players.

## Checklist

- [x] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

heinrich5991 commented:
Same for econ ports and `bindaddr`s, could you add that as well?

> Having a `reload` read that in would be horrible ux for connected players.

I mean, it could be implemented using redirects, but I don't think it's worth it.

ChillerDragon commented:
> Same for econ ports and bindaddrs, could you add that as well?

Sure added it. But I also now moved the responsibility to you :p 
The ``ec_bindaddr`` and ``ec_port`` are config only already I realized. They can not be set from the rcon, because they do not have the server flag.
And ``binaddr`` I now added.

> I mean, it could be implemented using redirects, but I don't think it's worth it.

Oh true. Not fully backwards compatible but could be cool. Also not sure if its worth it. Until then we keep it read only. If someone wants to pick it up it can be reverted quickly.

heinrich5991 reviewed (approved):
Thanks.
